electron/package.json

```diff
@@ -3,15 +3,15 @@
     "version": "1.0.0-rc.1",
     "main": "dist/main.js",
     "scripts": {
-        "test": "mocha dist/**/*.spec.js"
+        "test": "./node_modules/.bin/_mocha -r ts-node/register ./src/**/*.spec.ts"
```

Comment:
They do, they will just be compiled during the test run, so you don't need to precompile them.
This way, debug breakpoints are returned to the IDE.